FRDeviceBinding/FRDeviceBindingTests/ApplicationPinDeviceAuthenticatorTests.swift

```diff
@@ -49,6 +49,8 @@ class ApplicationPinDeviceAuthenticatorTests: FRBaseTestCase {
             XCTAssertEqual(messageDictionary[DBConstants.sub] as? String, userId)
             XCTAssertEqual(messageDictionary[DBConstants.exp] as? Int, Int(expiration.timeIntervalSince1970))
             XCTAssertEqual(messageDictionary[DBConstants.platform] as? String, DBConstants.ios)
+            XCTAssertGreaterThan(messageDictionary[DBConstants.iat] as! Int, Int(Date().timeIntervalSince1970) - 10)
```

Comment:
will this cause any flakky test ? just making sure,  may be another idea is to use XCTAssertEqualWithAccuracy

Reply:
@jeyanthanperiyasamy Good suggestion, changed to use XCTAssertEqualWithAccuracy